test/erllambda_logging_handler.erl

```diff
@@ -11,4 +13,6 @@ handle(_Event, _Context) ->
     erllambda:message("Starting logs test"),
     erllambda:message("This line should not break logging", []),
     erllambda:message("This and any further logs should be printed"),
+    ?LOG("Using LOG macro should give module and line numbers in the logs"),
```

Comment:
Commenting here just to make sure this example is not merged by accident.

Reply:
What is wrong with this example?

Reply:
please disregard 🤦‍♂️ I confused it with a different module.